src/storage/src/source/kafka.rs

```diff
@@ -1661,92 +1660,87 @@ fn render_metadata_fetcher<G: Scope<Timestamp = KafkaTimestamp>>(
     (metadata_stream, probe_stream, button.press_on_drop())
 }
 
-fn spawn_metadata_thread<C: ConsumerContext>(
+fn spawn_metadata_task<C: ConsumerContext>(
```

Comment:
The metadata thread is now a tokio task. I'm not sure why it was a thread before. Perhaps it's from a time where we didn't have async support in Timely operators?